data-model-for-portal/Microsoft.DocumentDB/cosmos-db.json

```diff
@@ -0,0 +1,296 @@
+            "help": {
+                "learnMore": {
+                    "url": "https://docs.microsoft.com/cli/azure/cosmosdb#az-cosmosdb-list"
+                },
+                "parameterSets": [
+                    {
+                        "parameters": [
+                            "[--resource-group]",
+                            "[--subscription]"
+                        ]
+                    }
+                ]
+            },
+            "examples": [
+                {
+                    "description": "List all Azure Cosmos DB database accounts in a resource group.",
+                    "parameters": [
+                        {
+                            "name": "--subscription",
+                            "value": "[path.subscriptionId]"
+                        },
+                        {
+                            "name": "--resource-group",
+                            "value": "[path.resourceGroupName]"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "az cosmosdb show",
+            "description": "Get the details of an Azure Cosmos DB database account.",
+            "path": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}",
+            "confirmation": false,
+            "help": {
+                "learnMore": {
+                    "url": "https://docs.microsoft.com/cli/azure/cosmosdb#az-cosmosdb-show"
+                },
+                "parameterSets": [
+                    {
+                        "parameters": [
+                            "[--ids]",
+                            "[--name]",
+                            "[--resource-group]",
+                            "[--subscription]"
+                        ]
+                    }
+                ]
+            },
+            "examples": [
+                {
+                    "description": "Get the details of an Azure Cosmos DB database account. (autogenerated)",
+                    "parameters": [
+                        {
+                            "name": "--subscription",
+                            "value": "[path.subscriptionId]"
+                        },
+                        {
+                            "name": "--name",
+                            "value": "[path.accountName]"
+                        },
+                        {
+                            "name": "--resource-group",
+                            "value": "[path.resourceGroupName]"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "az cosmosdb database list",
+            "description": "Lists all Azure Cosmos DB databases.",
+            "path": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/listKeys",
```

Comment:
Seems the path is not correct, please double check.

Reply:
I have checked the path of the command again. It seems that this command uses this unusual path. <img width="858" alt="image" src="https://user-images.githubusercontent.com/79819545/223902122-badb81d1-bc89-42b5-907a-d7bcd0a8f322.png">

Reply:
there are other http requests in command, 

![image](https://user-images.githubusercontent.com/16348853/224632198-4d2e5b96-017f-4445-829c-abd2e42ff188.png)

Reply:
Received, already modified.